rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -3479,5 +3482,166 @@ namespace rsx
 				intr_thread->cmd_notify.notify_one();
 			}
 		}
+
+		const u64 current_time = get_system_time();
```

Comment:
This new code shouldn't even go into one function. Its almost 200 lines long. Move it somewhere else.

Reply:
Fixed.